src/_tkagg.cpp

```diff
@@ -232,6 +264,90 @@ static PyMethodDef functions[] = {
     { NULL, NULL } /* sentinel */
 };
 
+#ifdef DYNAMIC_TKINTER
+// Functions to fill global TCL / Tk function pointers from tkinter module.
+
+#include <dlfcn.h>
```

Comment:
This include fails when using VS

```
src/_tkagg.cpp(270) : fatal error C1083: Cannot open include file: 'dlfcn.h': No such file or directory
```

https://ci.appveyor.com/project/mdboom/matplotlib/build/1.0.1545/job/724qv5hcfngqa2r9#L1033